instance/models/openedx_instance.py

```diff
@@ -198,6 +199,25 @@ def database_name(self):
         escaped = ''.join(char for char in name if char in allowed)
         return truncate_name(escaped, length=50)
 
+    @property
+    def shut_down(self):
```

Comment:
The naming of this property could use some clarification, especially if, as per @antoviaque's request, we're adding a `shutdown()` method to the instance.